### PR TITLE
Add regenerate bundle function and tests

### PR DESCRIPTION
**Pull request by pbortlov**
* add regenerate_bundle function to iib_client.py
* add tests to test_iib_client.py

### CODE REVIEW

**Comment by pbortlov** [comment]
@yashvardhannanavati @chandwanitulsi PTAL

File: iiblib/iib_client.py [hunk 1]
```diff
@@ -292,6 +297,43 @@ def wait_for_build(self, build):
...
+        Returns:
+            `RegenerateBundleModel` or dict
+              if raw == True return dict with json response otherwise
+              return `RegenerateBundleModel` instance.
+        """
+
+        post_data = {
+            "from_bundle_image": bundle_image,
+        }
+        if organization:
```
Comment (lipoja) [inline iiblib/iib_client.py:327]: Is organization mandatory parameter for this API endpoint? If yes we can remove this if. 
If it is optional we should make the parameter optional as well. `organization=None` L303

Comment (pbortlov) [reply iiblib/iib_client.py:327]: Thank you. I missed that. Fixed

**Comment by lipoja** [comment]
@pbortlov I think it looks ok, however we should bump the version so this PR pass the CI test

**Comment by pbortlov** [comment]
> @pbortlov I think it looks ok, however we should bump the version so this PR pass the CI test

I thought to add a commit with bumping a version and add a tag. What do you think @lipoja?

**Comment by lipoja** [comment]
> > @pbortlov I think it looks ok, however we should bump the version so this PR pass the CI test
> 
> I thought to add a commit with bumping a version and add a tag. What do you think @lipoja?

Ideally yes, however I see everything red, that merging is blocked because of the failing CI. 
Let's make it green for now and think about this version checking. Maybe we can turn in on just for master branch (run this check after merge).

**Comment by pbortlov** [comment]
> > > @pbortlov I think it looks ok, however we should bump the version so this PR pass the CI test
> > 
> > 
> > I thought to add a commit with bumping a version and add a tag. What do you think @lipoja?
> 
> Ideally yes, however I see everything red, that merging is blocked because of the failing CI.
> Let's make it green for now and think about this version checking. Maybe we can turn in on just for master branch (run this check after merge).

I change the setup.py and add a comment to the changelog. It should be fine now

**Review by yashvardhannanavati — DISMISSED** [review]

**Comment by yashvardhannanavati** [comment]
Let's fix the pipeline and then merge it. :)

File: setup.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def read_content(filepath):
 
 setup(
     name="iiblib",
-    version="2.0.1",
+    version="2.0.2",
```
Comment (lipoja) [inline setup.py:65]: The version should be 2.1.0

Comment (pbortlov) [reply setup.py]: fixed

**Review by lipoja — DISMISSED** [review]

**Review by querti — APPROVED** [review]

**Review by midnightercz — APPROVED** [review]

File: docs/source/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+ - regenerate_bundle function in iib_client.py
+ - regenerate_bundle test in test_iib_client.py
```
[the remark this answers is not in the document]
Comment (pbortlov) [reply docs/source/CHANGELOG.md:11]: ok

**Review by lipoja — APPROVED** [review]

**Review by midnightercz — APPROVED** [review]

**Review by yashvardhannanavati — APPROVED** [review]

**Pull request merged**